src/uk/gov/defra/ffc/Tests.groovy

```diff
@@ -111,4 +111,27 @@ class Tests implements Serializable {
     'SONAR_PR_REPOSITORY': "defra/${projectName}"
     ];
   }  
+
+  static def runAcceptanceTests(ctx, pr) {
```

Comment:
Just remembered about the GitHub status check. I think it would be worth adding one to this step in the same way as can be seen in `lintHelm` https://github.com/DEFRA/ffc-jenkins-pipeline-library/pull/156/files#diff-c59dffe0a9d707979f83d6c34944c959R26

Reply:
I've added the status check wrapper

Reply:
This works well. One possible improvement would be to move the status wrapper inside of the if check so it is only added if the acceptance tests are run. What do you think?

Reply:
Yep agreed & done